templates/keyval.epp

```diff
@@ -0,0 +1,3 @@
+<% $kvlist.keys.sort.each | $k | { -%>
```

Comment:
please add a header to the template where it accepts the passed in parameters.

Reply:
How could i add a header and not change file content (and service restart) ?

Reply:
@phaedriel see https://www.puppet.com/docs/puppet/7/lang_template_epp.html#epp-parameter-tags

On a side note I saw a lot of closed PR / opened PR.  You do not need to open a new PR when you want to make some change: PRs are "linked" to your branch, so if you push new commit or rewrite history and force push, the PR is updating accordingly :wink:

Reply:
Thanks !

I try merge master on my branch and rebase but my push contained "modulesync modification". Probably don't use correct git commands. Sorry